exec/jdbc/src/main/java/org/apache/drill/jdbc/impl/DrillCursor.java

```diff
@@ -376,6 +417,19 @@ synchronized void cleanup() {
     currentBatchHolder.clear();
   }
 
+  long getTimeoutInMilliseconds() {
+    return timeoutInMilliseconds;
+  }
+
+  //Set the cursor's timeout in seconds
+  void setTimeout(int timeoutDurationInSeconds){
+    this.timeoutInMilliseconds = timeoutDurationInSeconds*1000L;
```

Comment:
Preferably use `TimeUnit.SECONDS.toMillis(timeoutDurationInSeconds)` to avoid magic constants

Reply:
+1